tests/contenttypes_tests/test_fields.py

```diff
@@ -33,3 +36,19 @@ def test_value_to_string(self):
         answer2 = Answer.objects.create(question=question)
         result = json.loads(Question.answer_set.field.value_to_string(question))
         self.assertCountEqual(result, [answer1.pk, answer2.pk])
+
+    def test_can_get_path_to_proxy_model(self):
```

Comment:
A higher level test (such as executing a query) is probably more appropriate. There aren't any unit tests at the `get_path_info()`  level as that's a private API subject to change. We don't want to have to figure out how to rewrite this test if it changes.